lib/backend/k8s/k8s.go

```diff
@@ -94,6 +95,9 @@ func NewKubeClient(kc *KubeConfig) (*KubeClient, error) {
 			*override.variable = override.value
 		}
 	}
+	if kc.K8sInsecureSkipTLSVerify {
```

Comment:
Ah I see, it's a bool and that bit above is just strings.  I suppose this is fine then.